src/main/java/com/rainbowrave/Interpolate.java

```diff
@@ -0,0 +1,20 @@
+package com.rainbowrave;
```

Comment:
This whole file holds 1 static function that is used in one place, I think it should be inlined to where it's used and this file deleted.

Reply:
Agreed, the original idea was the possibility of adding other interpolation styles that could be chosen in the options, but this ended up being unnecessary.